app/concepts/data_source/stock/tribunal_instance/unit/operation/load_transmission.rb

```diff
@@ -5,6 +5,36 @@ module Unit
         module Operation
           class LoadTransmission < Trailblazer::Operation
 
+            step Nested(ZIP::Operation::Extract)
+            fail :log_nested_error, fail_fast: true
+            step :check_file_count
+            fail :too_many_files, fail_fast: true
+            step ->(ctx, extracted_files:, **) { ctx[:extracted_file] = extracted_files.first }
+            pass :log_zip_info
+            step Nested(Import)
+            fail :log_nested_error
+
+            # TODO: find a better solution ?
+            step ->(ctx, dest_directory:, **) { FileUtils.rm_rf(dest_directory) }
```

Comment:
Normalement possible avec Trailblazer 2.1 : il y a une option `group: :end` ou un truc du genre pour exécuter une tâche en fin d'opération, qu'elle finisse en failure ou success (j'ai jamais testé encore)

Reply:
je vais regarder

Reply:
http://trailblazer.to/2.1/trailblazer/wiring.html#doormat-step-group
C'est à ça que je pensais, ça fait pas exactement ce que je pensais mais ça reste utilisable dans ton cas même si c'est pas parfait.

Reply:
je n'ai pas l'impression que ça serve à ça.
C'est pour les [template d'opération](http://trailblazer.to/api-docs/#operation-dsl-group) j'ai fais queqlues tests non concluant.

Si pas mieux, ça reste ainsi pour le moment